tests/test_auth_basic_auth.py

```diff
@@ -93,15 +98,11 @@ def test_valid_cookie(self):
         assert_equal(res.status_code, 200)
 
     def test_expired_cookie(self):
-        self.session = SessionFactory(user=self.user1)
-        with connection.cursor() as cursor:
-            cursor.execute("""
-                UPDATE osf_session
-                SET created = %s
-                WHERE id = %s
-            """, [(timezone.now() - timedelta(seconds=settings.OSF_SESSION_TIMEOUT)), self.session.id])
         cookie = self.user1.get_or_create_cookie()
+        session_key = UserSessionMap.objects.filter(user=self.user1)[0].session_key
+        session = SessionStore(session_key=session_key)
+        session.delete()
         self.app.set_cookie(settings.COOKIE_NAME, str(cookie))
         res = self.app.get(self.reachable_url)
         assert_equal(res.status_code, 302)
-        assert_in('login', res.location)
+        assert_equal('http://localhost/', res.location)
```

Comment:
Ah, with Django session, the expiration is no longer controlled by us, so we have to use "deletion" to simulate the "expired" case. Is it OK to access DB and modify the expiration directly if we decide to use Django DB as the testing backend?

Reply:
Maybe? You can make a PR to change this if you wish.